### DISCUSSION TITLE
Anvil upgrade

### DISCUSSION THREAD

**Discussion by P3pp3rF1y** [P3pp3rF1y/SophisticatedBackpacks#86] [Ideas]
Just a simple tab with similar gui to vanilla anvil

**Comment by TheDeviantCrafter** [comment]
Personally, I would prefer if backpacks focused on tasks that are useful when exploring or mining. Anvils are usually used in a dedicated enchanting area. Building one of those is more engaging than simply slapping an upgrade in a UI.

**Reply by P3pp3rF1y** [reply]
The primary reason for anvil that I am thinking of here is repairing your items on the go, it's not really because the primary motivation would be somehting like applying enchanted books or combining items. I believe people will rarely use this upgrade for enchanting because as you said you need to have dedicated area for that anyway and then having portable anvil doesn't bring that much benefit, but I believe that unless you have mending on your items it can be pretty useful to be able to repair your pickaxe in a mine without the need to carry and set down anvil every time.
